src/_pytest/main.py

```diff
@@ -357,7 +359,7 @@ class Failed(Exception):
 
 @attr.s
 class _bestrelpath_cache(dict):
-    path = attr.ib()
+    path = attr.ib(type=py.path.local)
```

Comment:
Hmm is this accepted? Does mypy understand `py.path.local` to be a proper type? (I'm asking because I remember I had some issue with that).

Reply:
Appears to be "Any" - will remove it.